.gitignore

```diff
@@ -231,6 +231,7 @@ node_modules/
 /ajax/libs/angular-gridster/v.0.10.4
 /ajax/libs/numbered/0.0.4
 /ajax/libs/numbered/1.0.0
+/ajax/libs/datatables-fixedheader/RELEASE*/
```

Comment:
I can't find `RELEASE` prefix, did I miss something?
https://github.com/DataTables/Dist-DataTables-FixedHeader/tags
But maybe it's fine here.
Because you met before.

Reply:
There is no versions with prefix in `datatables-fixedheader`. I think it would be better to remove https://github.com/cdnjs/cdnjs/pull/11532/commits/4fd5a0fbc265f2eeeffbc22faf41c0624ac91502. Sorry for not noticing last time. Maybe you saw other library like  `datatables` or `datatables-colvis`? Or the author could have deleted tags.